src/store/student-store.js

```diff
@@ -82,4 +107,7 @@ store.registerHandler('CHECKIN', data => {
     student.items.splice(student.items.indexOf(item), 1);
 });
 
+store.registerHandler('UPDATE_STUDENT', student => {
+    updateStudent(student.id, student.name, student.email, student.major);
+});
```

Comment:
That's the problem. There was an empty line between the handlers and the export, and now there is not.